hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -413,67 +413,107 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		hcluster.Status.Version = computeClusterVersionStatus(r.Clock, hcluster, hcp)
 	}
 
-	// Set the ClusterVersionSucceeding based on the hostedcontrolplane
+	// Copy the CVO conditions from the HCP.
+	hcpCVOConditions := map[hyperv1.ConditionType]*metav1.Condition{
+		hyperv1.HostedControlPlaneDegraded:    meta.FindStatusCondition(hcp.Status.Conditions, string(hyperv1.HostedControlPlaneDegraded)),
+		hyperv1.ClusterVersionProgressing:     meta.FindStatusCondition(hcp.Status.Conditions, string(hyperv1.ClusterVersionProgressing)),
+		hyperv1.ClusterVersionReleaseAccepted: meta.FindStatusCondition(hcp.Status.Conditions, string(hyperv1.ClusterVersionReleaseAccepted)),
+		hyperv1.ClusterVersionUpgradeable:     meta.FindStatusCondition(hcp.Status.Conditions, string(hyperv1.ClusterVersionUpgradeable)),
+	}
+	for conditionType := range hcpCVOConditions {
+		var hcCVOCondition *metav1.Condition
+		// Set unknown status.
+		hcCVOCondition = &metav1.Condition{
+			Type:               string(conditionType),
+			Status:             metav1.ConditionUnknown,
+			Reason:             hyperv1.StatusUnknownReason,
+			Message:            fmt.Sprintf("The hosted control plane is not found: %v", err),
```

Comment:
Same as above, the message is incorrect if err == nil && hcpCVOConditions[conditionType] == nil